test/stripe/stripe_object_test.rb

```diff
@@ -3,10 +3,13 @@
 module Stripe
   class StripeObjectTest < Test::Unit::TestCase
     should "implement #respond_to correctly" do
-      obj = Stripe::StripeObject.construct_from({ :id => 1, :foo => 'bar' })
+      obj = Stripe::StripeObject.construct_from({ :id => 1, :foo => 'bar', :qux => true })
       assert obj.respond_to?(:id)
       assert obj.respond_to?(:foo)
       assert !obj.respond_to?(:baz)
+      assert obj.respond_to?(:qux)
+      assert obj.respond_to?(:qux?)
+      assert !obj.respond_to?(:foo?)
```

Comment:
+1 to these test changes. What do you think of one more test to check that a boolean method does indeed work outside of `respond_to?`?